rasa/utils/tensorflow/model_data.py

```diff
@@ -709,6 +709,13 @@ def balanced_data(self, data: Data, batch_size: int, shuffle: bool) -> Data:
         ):
             return data
 
+        if balance_once:
```

Comment:
This might be not ideal, because the lowest batch size(provided there is a range) is used to calculate initial quotas for each class